package.json

```diff
@@ -224,6 +226,7 @@
     "maildev": "^2.1.0",
     "mockdate": "^3.0.5",
     "prettier": "^3.3.3",
+    "react-dom": "^18.3.1",
```

Comment:
Trying to understand the usage of react-dom as well - I'm supposing react-dom is used for the stories and `index.tsx`

Reply:
Rationale: 
- react-email used to have react-dom as a dependency in v2.x.x but they switched it to a peer dependency in v3.0.2 (react-email uses react-dom internally)

peer dependency means it is the responsibility of the 'peer' = aka FormSG in this case to install it as a dependency -> hence we add `react-dom` to our dependencies. 

This change makes sense now